### PR TITLE
CT-1354 manage api setup automation

### PR DESCRIPTION
**Pull request by martinjandl**
Jira: [CT-1354](https://keboola.atlassian.net/browse/CT-1354)
Connection PR: XXXXX

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

## Release

- [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

---

[CT-1354]: https://keboola.atlassian.net/browse/CT-1354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,52 +86,75 @@ KBC_TEST_ADMIN_TOKEN=token_of_another_admin_having_mfa_disabled
...
+
+#### Prerequisites:
+
+- configured and logged in az, aws and gcp CLI tools
+- installed terraform (https://www.terraform.io) and jq (https://stedolan.github.io/jq) to setup local env
+
+```shell
+
+# set environment variables
+export NAME_PREFIX=<your_nick> # your name/nickname to make your resource unique & recognizable
```
Comment (jirkasemmler) [inline README.md:115]: btw pro ten name prefix by tam mela byt poznamka, ze tam maji byt jen azAZ09-

Comment (jirkasemmler) [reply README.md:115]: sam jsem na tom vyhorel

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
par malickosti, zejmena naming

File: README.md [hunk 2]
```diff
@@ -86,52 +86,75 @@ KBC_TEST_ADMIN_TOKEN=token_of_another_admin_having_mfa_disabled
...
+export AZURE_SUBSCRIPTION_ID=<your_subscription_id> # Azure subscription id
+
+# persist variables to terraform
+cat <<EOF > ./provisioning/terraform.tfvars
+name_prefix = "${NAME_PREFIX}"
+aws_profile = "${AWS_PROFILE}"
+aws_account = "${AWS_ACCOUNT}"
+aws_region = "${AWS_REGION}"
+gcp_project_id = "${GCP_PROJECT_ID}"
+gcp_project_region = "${GCP_PROJECT_REGION}"
```
[the remark this answers is not in the document]
Comment (martinjandl) [reply README.md:133]: Tady to chapu tak, ze jsou dva ruzne gcp regiony GCP_REGION a GCP_PROJECT_REGION, potom mam v tfvars napr.
gcp_storage_location = "us-central1"
gcp_project_region = "europe-west1"

File: README.md [hunk 3]
```diff
@@ -86,52 +86,75 @@ KBC_TEST_ADMIN_TOKEN=token_of_another_admin_having_mfa_disabled
...
 ```
 
-Source newly created file and run tests:
-
+Run tests
 ```bash
 docker-compose run --rm dev composer tests
 ```
 
-### Required variables
```
Comment (jirkasemmler) [inline README.md:122]: tady to popsani variables by nemelo zmizet. asi bych ho mergnul s tim co je na lajne 75. To vysvetleni je tam dulezite

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jirkasemmler) [reply README.md]: ok, asi muze byt. prislo mi to jako typo

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
jeste jsem si tam vsiml jedne malickosti, tak na to prosim mrkni. jinak ok a frci to

File: README.md (hunk 3 above)
Comment (martinjandl) [reply README.md:122]: jo, jasne, to je chyba, prehledl jsem

File: README.md [hunk 4]
```diff
@@ -72,66 +72,81 @@ so you can disable this by creating new file `phpunit-retry.xml` from `phpunit-r
...
-TEST_S3_KEY=
-TEST_S3_REGION=
-TEST_S3_SECRET=
-TEST_GCS_KEYFILE_JSON=
-TEST_GCS_KEYFILE_ROTATE_JSON=
-TEST_GCS_FILES_BUCKET=
-TEST_GCS_REGION=
+KBC_MANAGE_API_SUPER_TOKEN_WITH_UI_MANAGE_SCOPE=super_token_with_ui_manage_scope # can be created in manage-apps on the Tokens tab. Token must have "ui manage" scope
+KBC_TEST_MAINTAINER_ID=id # `id` of maintainer. Please create a new maintainer dedicated to test suite. All maintainer's organizations and projects all purged before tests!
+KBC_TEST_ADMIN_EMAIL=email_of_another_admin_having_mfa_disabled # email address of another user without any organizations
```
Comment (tomasfejfar) [inline README.md:84]: KBC_TEST_ADMIN_EMAIL=devel-tests+test_admin@keboola.com

Would work as well

**Pull request merged**